lessons/advanced/protocols.md

```diff
@@ -110,6 +110,6 @@ iex> to_atom(%{foo: "bar"})
 :foo
 ```
 
-It is worth nothing that although underneath structs are Maps, but they do not share protocol implementations with Maps. They are not enumerable, they cannot be accessed.
+It is worth noting that although underneath structs are Maps, but they do not share protocol implementations with Maps. They are not enumerable, they cannot be accessed.
```

Comment:
@doomspork I didn't have any issues with understanding that sentence but as I mentioned above this option could prevent potential misreading that led to creation of this issue.